test/fixtures/tvos_app/bwx_spec.json

```diff
@@ -419,5 +419,6 @@
             "swiftmodules": [],
             "test_host": null
         }
-    ]
+    ],
+    "scheme_autogeneration_mode": "auto"
```

Comment:
Hmm. Why is it still showing at the bottom here? Do the fixtures need regenerating?

Reply:
Sorry. I forgot to regenerate the fixtures after updating.